hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaConsumerGroupManagerSpec.groovy

```diff
@@ -0,0 +1,131 @@
+package pl.allegro.tech.hermes.management.infrastructure.kafka.service
+
+import org.apache.kafka.clients.admin.AdminClient
+import org.apache.kafka.clients.admin.NewTopic
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.TopicPartition
+import org.junit.Rule
+import org.springframework.boot.test.rule.OutputCapture
+import org.testcontainers.containers.KafkaContainer
```

Comment:
Wow, nice that you decided to use here Kafka container. In the future we would like to rewrite all hermes integration tests to use containers as well (#1072 ).